### DISCUSSION TITLE
Workspace Notes for Different Collections?

### DISCUSSION THREAD

**Discussion by cyanidearsenic** [windingwind/zotero-better-notes#653] [Q&A] [answered]
Is it possible to have a different workspace notes for different collections in Zotero? 
I've been using my workspace note as a sort of bibliography of notes I've taken on collection items from my main collection like how in the image I have it set, so I can use the outline to quickly find the notes I took on chapters of this book (sorry if the contrast is hard on your eyes, I use dark mode and I can't seem to find any way to change the colour of the text on the outline).
![image](https://github.com/windingwind/zotero-better-notes/assets/12106286/f6a75bc0-51f5-4296-8302-8f83b1730255)
But now I have a secondary collection for research on an unrelated topic, so I was wondering if it was possible to have a separate workspace note per collection or do I need to just try and find a way to tidily divide my main workspace note, so that the outline is easily used for finding specific pages of notes in either collection.

**Comment by windingwind** [comment, marked as answer]
The darkmode bug should be reported here: https://github.com/windingwind/zotero-better-notes/issues

You can manually switch to different workspace notes via the menu - File - Open Recent Workspace Note.

**Reply by cyanidearsenic** [reply]
Oh, I didn't realise that was a darkmode bug, I thought it was just how it had been meant to look but I'll go report it now.
Thanks
